phpBB/assets/javascript/plupload.js

```diff
@@ -446,6 +446,44 @@ phpbb.plupload.fileError = function(file, error) {
 phpbb.plupload.uploader = new plupload.Uploader(phpbb.plupload.config);
 phpbb.plupload.initialize();
 
+/**
+ * Add a file filter to check for max file sizes per mime type.
+ */
+plupload.addFileFilter('mime_types_max_file_size', function(types, file, cb) {
```

Comment:
```suggestion
plupload.addFileFilter('mime_types_max_file_size', function(types, file, callback) {
```
Please rename all occurences to `callback`. There is just no point in calling it `cb` unless one wants to reduce readability.